lms/static/sass/views/_login-register.scss

```diff
@@ -111,51 +97,35 @@ $sm-btn-linkedin: #0077b5;
 
             .text {
                 position: relative;
-                background: white;
-                padding: 0 10px;
-                z-index: 6;
+                top: -2px; // Aligns center of text with center of line (CR)
+                padding: 0 $baseline;
+                background: $white;
                 text-transform: none;
```

Comment:
Let's move this text-transform reset to the element it is applied to: the H2 above.